### PR TITLE
Fix compatibility with phpstan 0.11

### LINKED ISSUE #51: The latest phpstan version is not supported
Phpstan 0.11 was released.

### PR DESCRIPTION
**Pull request by jakzal**
Fixes #51

### CODE REVIEW

File: src/Rules/ThrowsPhpDocInheritanceRule.php [hunk 1]
```diff
@@ -66,11 +68,14 @@ public function getNodeType(): string
 	}
 
 	/**
-	 * @param ClassMethod $node
```
Comment (jakzal) [inline src/Rules/ThrowsPhpDocInheritanceRule.php:69]:
```
Parameter #1 $node (PhpParser\Node\Stmt\ClassMethod) of method
Pepakriz\PHPStanExceptionRules\Rules\ThrowsPhpDocInheritanceRule::proce
ssNode() should be contravariant with parameter $node (PhpParser\Node)
of method PHPStan\Rules\Rule::processNode()
```

File: src/Rules/ThrowsPhpDocInheritanceRule.php [hunk 2]
```diff
@@ -102,11 +107,9 @@ public function processNode(Node $node, Scope $scope): array
 		}
 
 		$throwType = $throwsTag->getType();
-		$parentClasses = $classReflection->getInterfaces();
-		$parentClass = $classReflection->getParentClass();
-		if ($parentClass !== false) {
-			$parentClasses[] = $parentClass;
-		}
```
Comment (jakzal) [inline src/Rules/ThrowsPhpDocInheritanceRule.php:109]: `Foreach overwrites $parentClass with its value variable.`

**Comment by pepakriz** [comment]
Wow, good job! Please change the branch-alias to `0.5-dev` and I'll make a new release.

**Comment by pepakriz** [comment]
I've changed it myself.

Thank you!

**Pull request merged**